omnigibson/simulator.py

```diff
@@ -230,8 +230,8 @@ class Simulator(lazy.omni.isaac.core.simulation_context.SimulationContext, Seria
         def __init__(
             self,
             gravity=9.81,
-            physics_dt=1.0 / 120.0,
-            rendering_dt=1.0 / 30.0,
+            physics_dt=1.0 / gm.DEFAULT_PHYSICS_FREQ,
+            rendering_dt=1.0 / gm.DEFAULT_RENDERING_FREQ,
```

Comment:
This is bad - it gets processed at import time so changing the macro does nothing. Can we default this into None and then do `if physics_dt is None: physics_dt = 1.0 / gm. ...` down the line

Reply:
fixed